app/views/games/index.html.erb

```diff
@@ -1,182 +1,51 @@
 <div class="container-fluid">
-  <section id="register">
-    <div class= "row">
+  <section id="games-list">
+    <div class="row">
       <div class="col-sm-5 col-sm-offset-1">
```

Comment:
This is just to offset 1 small column from left side of page. Is this not a good practice? Find myself using it frequently to keep things off the edges...